### DISCUSSION TITLE
Multi row / Batch writes to Postgres using JDBC

### DISCUSSION THREAD

**Discussion by dashingsat** [micronaut-projects/micronaut-data#889] [Q&A]
Hello , Is there a way , in which we can use Micronaut data JDBC variant to make batch ( Multi row ) writes in Postgres ?

**Comment by raghu298** [comment]
jdbc batchupdate

**Reply by graemerocher** [reply]
See https://micronaut-projects.github.io/micronaut-data/latest/guide/#updates

**Reply by raghu298** [reply]
@graemerocher  didnot get you?
Is there anything like spring batch in micronaut where we can define the pipeline of steps and can have reader, writer configuration and all?

**Reply by graemerocher** [reply]
no there is nothing like spring batch and no plans to create one

**Reply by raghu298** [reply]
@graemerocher How to achieve batch jobs in micronaut?
or can we integrate spring-batch into micronaut?
